app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/welcome/WikiEducationDashboardFragment.kt

```diff
@@ -56,8 +56,8 @@ class WikiEducationDashboardFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.fragment_wiki_edu_dashboard, container, false)
-        cv_signup_wikipedia = view.findViewById(R.id.cv_signup_wikipedia)
-        cv_login_wikipedia = view.findViewById(R.id.cv_login_wikipedia)
+        cv_signup_wikipedia = view.findViewById(R.id.cvSignUpWikipedia)
+        cv_login_wikipedia = view.findViewById(R.id.cvLoginWikipedia)
```

Comment:
not needed.

Reply:
this PR is for outreachProgramsDashboardFragment.kt